docs/config.json

```diff
@@ -368,8 +368,8 @@
       "timeline": [
         {
           "moved": 0,
```

Comment:
The README isn't clear, am I supposed to do something with this value? I think it's supposed to show the movement, should this be `-1` like AWS ELB?